cmd/setup.go

```diff
@@ -180,38 +181,52 @@ func configureMessengers(conf messagingConfig, cache *util.Cache) chan push.Even
 	return notificationChan
 }
 
-func configureTariffs(conf tariffConfig) (t api.Tariff, err error) {
+func configureTariffs(conf tariffConfig) (tariff.Tariffs, error) {
+	var grid, feedin api.Tariff
+	var currencyCode currency.Unit = currency.EUR
+	var err error
+
+	if conf.Currency != "" {
```

Comment:
warum brauchen wir die Currency überhaupt als etwas anderes als String?

Reply:
Konfigurationsfehler vermeiden. Wir reichen den Wert ja für die Geld-Formatierung in die UI hoch. Wenn da quatsch drin steht bekommen wir Fehler in der UI. Da es den currency.Unit Typ gibt würd ich den hier benutzen. Free-Error-Checking.